cloudformation/events-cloudformation.json

```diff
@@ -42,84 +75,7 @@
               "ec2.amazonaws.com"
             ],
             "eventName": [
-              "CreateBucket",
```

Comment:
Please note that event manually creating an EventBridge rule with the event pattern prescribed here fails with:

> Parameter EventPattern for rule jupiterone-cloudtrail-events exceeds limit of 2048.

even though the `EventPattern` appears to be 1.61kb. I ripped out most of the events in order to shrink the size for testing purposes.